formats/tokens.js

```diff
@@ -0,0 +1,28 @@
+const path = require('path');
+
+const colorFactory = require('./utils/color-factory/color-factory');
+const {tokensToJson} = require('./utils/color-factory/utils');
+
+function tokenify(scheme) {
+  return (result) => {
+    const configFile = path.basename(result.toJS().meta.file, '.yml');
+    let configArg;
+
+    try {
```

Comment:
Going to switch this to use `fs` and check if the file exists instead of a try catch block, I think that’s a bit cleaner.